### DISCUSSION TITLE
Aggregators assigning incorrect taxonomy

### DISCUSSION THREAD

**Discussion by campmlc** [ArctosDB/arctos#8043] [General]
Just discovered that records in Arctos identified as subclass Acari, a legacy name for ticks and mites accepted by WoRMS and TPT via Arctos sources, are being converted by GBIF to a butterfly genus and by iDigBio to a Mollusc. 
Is there some way to correct this via the aggregators? 

https://arctos.database.museum/guid/MSB:Para:44464
See https://www.gbif.org/occurrence/4165998931
https://www.idigbio.org/portal/records/1faf948d-9428-449a-9493-22bc9d1b623c

But GloBI gets it right: https://www.globalbioticinteractions.org/?accordingTo=http%3A%2F%2Farctos.database.museum%2Fguid%2FMSB%3APara%3A44464&interactionType=interactsWith

FWIW, mite classification is a mess. We are dealing with field IDs and rough sorts of ethanol preserved or frozen lots, so we can't assign to an order. The alternative is to call them all "Arachnida" which is a disservice to parasitologists.

**Comment by DerekSikes** [comment]
Strange. I checked one of our records with ID = Acari and it was not converted to a Lepidopteran: https://www.gbif.org/occurrence/1146095272

It was converted to 'Arachnida'

but yours, with the same ID was converted: https://www.gbif.org/occurrence/4165998931

**Comment by dustymc** [comment]
>same ID

No.

<img width="595" alt="Screenshot 2024-06-14 at 08 41 32" src="https://github.com/ArctosDB/arctos/assets/5720791/0e2cbe92-d29f-4d51-b124-a56cedc6d948">
<img width="597" alt="Screenshot 2024-06-14 at 08 41 23" src="https://github.com/ArctosDB/arctos/assets/5720791/7ec918f0-61eb-4835-b15f-1005a716cf1e">





<img width="1082" alt="Screenshot 2024-06-14 at 08 44 03" src="https://github.com/ArctosDB/arctos/assets/5720791/1b5f52cf-a054-4e06-ba2f-a5a47cd7073a">
<img width="1140" alt="Screenshot 2024-06-14 at 08 43 53" src="https://github.com/ArctosDB/arctos/assets/5720791/2a0eaaad-48bc-4340-bae9-58546d310bb3">

**Comment by campmlc** [comment]
Yes, looks like UAM:Ento is preferring the Arctos classification, and MSB:Para is using WoRMS and TPT. Both classifications are identical except Arctos adds the order and WoRMS adds the subphylum, which honestly should be even clearer that neither of these are lepidopterans or molluscs.

**Comment by campmlc** [comment]
We had a similar problem a number of years ago when GBIF was converting all of our Class Aves birds to genus Avus beetle.
Here they are converting the subclass Acari to the genus Acara. I'd like to at least get the GBIF records fixed. Who do we talk to?

**Comment by mkoo** [comment]
> We had a similar problem a number of years ago when GBIF was converting all of our Class Aves birds to genus Avus beetle. Here they are converting the subclass Acari to the genus Acara. I'd like to at least get the GBIF records fixed. Who do we talk to?

@dbloom can you help connect us to the appropriate GBIF contact?

**Comment by Jegelewicz** [comment]
This is an issue at GBIF, not the IPT. I suggest reporting through https://github.com/gbif/portal-feedback/issues

**Comment by ekrimmel** [comment]
IMO reporting to GBIF, like Teresa suggests, is a good idea. From dealing with similar problems in other settings, it is particularly painful to fix an incorrect interpretation like this when the value in _dwc:scientificName_ does not have a DwC term to map to (e.g. Acari is a subclass but there is no term for _dwc:subclass_). It may be worth considering:

- Does Arctos publish data to _[dwc:taxonRank](https://dwc.tdwg.org/terms/#dwc:taxonRank)_? I think GBIF uses this in their data interpretation processing. At the very least, having the rank show up on the occurrence record in the "Original" data might help alert a user to the fact that (in this case) MSB is right and GBIF is wrong. 
- For names coming from WoRMS, Arctos knows the AphiaID. It would be worth publishing the URI (i.e. "urn:lsid:marinespecies.org:taxname:292684") via _[dwc:scientificNameID](https://dwc.tdwg.org/terms/#dwc:scientificNameID)_. I don't know if GBIF would use this but it certainly seems like a no brainer to match stuff based on identifiers vs. strings.

Based on [this DwC mapping Google Sheet](https://docs.google.com/spreadsheets/d/1IeqgpGaj7DkzHSNPrbTLpMdWSEx3IlYwu3xAYjRpec8/edit?gid=0#gid=0) it looks to me like Arctos thinks it is publishing data in both of the above DwC fields, but I don't see any values present on the DwC-A version of Mariel's record, even though Arctos clearly has information for both. On Derek's record there is a value for _dwc:taxonRank_, so maybe Arctos is making decisions based on classification source or on what the rank is (subclass-Mariel vs. order-Derek).

Another note, GBIF is currently using its own taxonomic "backbone" but will transition to using Catalog of Life sometime in the relatively near future. COL also does not know about Acari, and [thinks Arachnida is a subclass](https://www.catalogueoflife.org/data/taxon/RC). So fixing problems now might not fix future problems if COL has yet a different (and not the same as yours) perception. I'm not sure why COL does not know about Acari, because they ingest WoRMS as a source, but if it is a legacy name then that's probably why. They also might be relying on a different taxonomic source for this part of the tree (some mythical canonical mite authority?).

**Comment by campmlc** [comment]
@dustymc can you address the feasibility of @ekrimmel 's suggestions?

Does Arctos publish data to [dwc:taxonRank](https://dwc.tdwg.org/terms/#dwc:taxonRank)? I think GBIF uses this in their data interpretation processing. At the very least, having the rank show up on the occurrence record in the "Original" data might help alert a user to the fact that (in this case) MSB is right and GBIF is wrong.
For names coming from WoRMS, Arctos knows the AphiaID. It would be worth publishing the URI (i.e. "urn:lsid:marinespecies.org:taxname:292684") via [dwc:scientificNameID](https://dwc.tdwg.org/terms/#dwc:scientificNameID). I don't know if GBIF would use this but it certainly seems like a no brainer to match stuff based on identifiers vs. strings.
